### PR TITLE
Add static data for testing and flesh out Event view

### PR DESCRIPTION
**Pull request by kiwiz**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ module.exports = {
     "rules": {
         "indent": [
             "error",
-            4
+            2
```
Comment (helenasw) [inline .eslintrc.js]: eh

File: assets/js/components/Event.jsx [hunk 2]
```diff
@@ -1,10 +1,26 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import Card from './spectre/Card'
 
-const Event = ({title}) => <div>{title}</div>
+const Event = ({title, subtitle, description, thumbnail}) => <Card>
```
Comment (helenasw) [inline assets/js/components/Event.jsx]: slight preference for enclosing in parens and putting `<Card>` on its own line

File: assets/js/components/Event.jsx [hunk 3]
```diff
@@ -1,10 +1,26 @@
...
+  <Card.Image>
+    <img className="img-responsive" src={thumbnail} />
```
Comment (helenasw) [inline assets/js/components/Event.jsx]: what's the point of having a `<Card.Image>` component if you can't just pass it a src and be done with it?

File: assets/js/components/Event.jsx [hunk 4]
```diff
@@ -1,10 +1,26 @@
...
+  </Card.Header>
+  <Card.Body>
+    {description}
+  </Card.Body>
+</Card>
 
 Event.propTypes = {
-    title: PropTypes.string.isRequired,
+  title: PropTypes.string.isRequired,
+  subtitle: PropTypes.string.isRequired,
```
Comment (helenasw) [inline assets/js/components/Event.jsx]: this probs doesn't need to be required

File: assets/js/components/EventList.jsx [hunk 5]
```diff
@@ -1,14 +1,27 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import Columns from './spectre/Columns'
+import Column from './spectre/Column'
```
Comment (helenasw) [inline assets/js/components/EventList.jsx]: I think you could also write this as:
```
import { Column, Columns } from './spectre'
```
no strong pref though

File: assets/js/components/EventList.jsx [hunk 6]
```diff
@@ -1,14 +1,27 @@
...
+        'col-sm-6',
+        'col-md-4',
+        'col-3',
+      ]}>
+        <Event
+          title={event.title}
+          subtitle={event.subtitle}
+          description={event.description}
+          thumbnail={event.thumbnail}
+          image={event.image}
```
Comment (helenasw) [inline assets/js/components/EventList.jsx]: why not just pass the `event` and let the `Event` component deal with knowing what's in an event?

File: assets/js/components/spectre/Checkbox.jsx [hunk 7]
```diff
@@ -4,18 +4,19 @@ import classNames from 'classnames'
...
 )
 
 Checkbox.propTypes = {
-    name: PropTypes.string,
-    checked: PropTypes.bool,
-    size: PropTypes.oneOf(InputSizes),
+  name: PropTypes.string,
+  checked: PropTypes.bool,
+  size: PropTypes.oneOf(InputSizes),
+  children: PropTypes.node,
```
Comment (helenasw) [inline assets/js/components/spectre/Checkbox.jsx:19]: is it normal/necessary to define this? every component has `props.children`

File: assets/js/components/spectre/Column.jsx [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+import classNames from 'classnames'
+
+const Column = (props) => (
+  <div className={classNames(
+    'column',
+    props.classNames
+  )}>
+    {props.children}
+  </div>
+)
```
Comment (helenasw) [inline assets/js/components/spectre/Column.jsx:12]: doesn't feel like you get much from this component

File: package.json [hunk 9]
```diff
@@ -26,6 +26,7 @@
     "prop-types": "^15.6.1",
     "react": "^16.2.0",
     "react-dom": "^16.2.0",
+    "react-google-maps": "^9.4.5",
```
Comment (helenasw) [inline package.json:29]: 🙀

**Review by helenasw — APPROVED** [review]

File: assets/js/components/Event.jsx (hunk 2 above)
Comment (kiwiz) [reply assets/js/components/Event.jsx]: :+1:

File: assets/js/components/Event.jsx (hunk 3 above)
Comment (kiwiz) [reply assets/js/components/Event.jsx]: :+1:

File: assets/js/components/Event.jsx (hunk 4 above)
Comment (kiwiz) [reply assets/js/components/Event.jsx]: Sanity check, mostly.

File: assets/js/components/spectre/Column.jsx (hunk 8 above)
Comment (kiwiz) [reply assets/js/components/spectre/Column.jsx:12]: :woman_shrugging:

File: assets/js/components/spectre/Checkbox.jsx (hunk 7 above)
Comment (kiwiz) [reply assets/js/components/spectre/Checkbox.jsx:19]: Just-in-case.

**Pull request merged**